### PR TITLE
Project Coding Convention added to README.md

### PR DESCRIPTION
**Pull request by guram21**
https://trello.com/c/pH2QcPOe

Please feel free to leave comments

### CODE REVIEW

**Review by Burdishka — APPROVED** [review]

**Review by Vadym11 — APPROVED** [review]

**Review by SergeiDemyanenko — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,80 @@
...
+com.project.selenium.tests
+com.project.selenium.utils
+```
+####Class names
+The upper camel case should be used for the class names. 
+The class name should be a noun. Try to make the name simple and descriptive.
+```
+LoginTest.java : All the tests shall have suffix Test
+LoginPage.java : All the pages shall have suffix Page
+TestBase.java : Class to include all common methods to be used by tests and pages
```
Comment (ke-de-ve) [inline README.md:27]: BaseTest.java  - we have test.entity.BaseTest

File: README.md [hunk 2]
```diff
@@ -1 +1,80 @@
...
+Meaningful directory name shall be used to indicate the content within the package.
+```
+com.project.selenium
+com.project.selenium.pages
+com.project.selenium.tests
+com.project.selenium.utils
+```
+####Class names
+The upper camel case should be used for the class names. 
+The class name should be a noun. Try to make the name simple and descriptive.
```
Comment (ke-de-ve) [inline README.md:23]: Name of the test class should include Test at the end. Name of the page class should include Page at the end.

File: README.md [hunk 3]
```diff
@@ -1 +1,80 @@
...
+```
+LoginTest.java : All the tests shall have suffix Test
+LoginPage.java : All the pages shall have suffix Page
+TestBase.java : Class to include all common methods to be used by tests and pages
+```
+####Method names
+A lower camel case should be used for the method names. 
+Method name should be a verb.
```
Comment (ke-de-ve) [inline README.md:31]: Method name should start with a verb  (do, is, make, change, etc).

File: README.md [hunk 4]
```diff
@@ -1 +1,80 @@
...
+```
+initialiseBrowsers()
+initialiseProperties()
+waitForTextToBePresent(WebElement webElement, String textToBeVerified)
+loginWithValidCredentialsTest() :All test methods shall end with suffix Test
+```
+####Variable names
+A lower camel case should be used for the variable names. 
+The intent of the variable shall be indicated with the given name. 
+The variable name should be a noun. Try to make the name simple and descriptive.
```
Comment (ke-de-ve) [inline README.md:43]: Try to make the name simple and descriptive using full words (button and not btn).

**Review by ke-de-ve — CHANGES_REQUESTED** [review]

**Review by ke-de-ve — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1 +1,90 @@
...
+The intent of the variable shall be indicated with the given name.<br> 
+The variable name should be a noun.<br> 
+Try to make the name simple and descriptive using full words.<br>
+For example: `button`, NOT `btn`.
+```
+webDriver
+еmployeeCount
+```
+####Constants
+Constants (final variables) must be all upper Snake cases, stylized as `SNAKE_CASE`.<br>
```
Comment (MaxBurdin) [inline README.md:55]: Constants are final **static** variables
https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html

**Review by MaxBurdin — CHANGES_REQUESTED** [review]

**Review by GalinaRuban — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1 +1,88 @@
...
+```
+####Variable names
+A lower camel case should be used for the variable names.<br> 
+The intent of the variable shall be indicated with the given name.<br> 
+The variable name should be a noun.<br> 
+Try to make the name simple and descriptive using full words.<br>
+For example: `button`, NOT `btn`.
+```
+actualValues
+rundomUser
```
Comment (MaxBurdin) [inline README.md:51]: r**a**ndomUser?

**Review by MaxBurdin — APPROVED** [review]

**Pull request merged**